lib/bundler.rb

```diff
@@ -159,12 +159,13 @@ def ruby_scope
     def user_home
       @user_home ||= begin
         home = Bundler.rubygems.user_home
+        dotbundle = home ? File.join(home, ".bundle") : nil
```

Comment:
This is super nitpicky but can we rename this to `bundle_home`. `dotbundle` doesn't really mean anything. :pray: